extension.Rmd

```diff
@@ -0,0 +1,241 @@
+
+```{r}
+
+plot(fitted(model_2), residuals(model_2),
+  xlab = "Fitted Values", ylab = "Residuals")
+  abline(h=0, lty=2)
+
+binnedplot(predict(model_2), resid(model_2), cex.pts=1, col.int="black")
+
+data_model2 <- data.frame(eu_neg = max(data$eu_neg, na.rm = T), z_dem_satis = max(data$z_dem_satis, na.rm = T), country = "AT") 
+
+predict(model_2, data_model2, type = "response")
+```
+
+
+For model 3, after plotting fitted values vs. residuals, we look for predicted probability for an individual whose context of unemployment rates, unemployment benefits, number of asylum seekers, and salience and variance of right-wing issues is Austria's most extreme--most likely to vote for an extreme right party, according to the theory.
+
+```{r}
+plot(fitted(model_3), residuals(model_3),
+  xlab = "Fitted Values", ylab = "Residuals")
+  abline(h=0, lty=2)
+
+binnedplot(predict(model_3), resid(model_3), cex.pts=1, col.int="black")
+
+#max_z_asylumseekers <- max(data$z_asylumseeker, na.rm = T) #4.45935
+#max_z_unemp <- max(data$z_unemp, na.rm = T) #12.2919
+#max_z_asylumseeker_z_unemp <- max(data$z_asylumseeker:data$z_unemp, na.rm = T) #-0.2445196
+#max_z_replacement <- max(data$z_replacement, na.rm = T) #32.96374
+#max_z_replacement_z_unemp <- max(data$z_replacement:data$z_unemp, na.rm = T) #0.5738534
+#max_z_replacement_z_asylumseeker <- max(data$z_replacement:data$z_asylumseeker, na.rm = T) #0.5738534
+
+#data_model3 <- data.frame(max_z_asylumseekers, max_z_unemp, max_z_asylumseeker_z_unemp, max_z_replacement, max_z_replacement_z_unemp, max_z_replacement_z_asylumseeker, country = "AT")
+#predict(model_3, data_model3, type = "response")
+#We have thus far been unable to debug the above code such that we could generate a prediction.  However, we do have a binned residual plot for this model.  
+```
+
+The binned residual plots show the highest level of randomness for the second and third models, Parsons' and the group conflict theories.  This method would indicate that, contrary to our conclusions from the separation plots (see below), these models have the closest fit to the observed data.  However, all of these conclusions must be qualified:  the proxy variables available to us in this data set do not perfectly operationalize the theoretical models in question.
+
+Conclusion: All Three Models
+
+![Model 1](figures/SP.Model1.png)
+
+![Model 2](figures/SP.model2.png)
+
+![Model 3](figures/SP.model3.png)
+
+
+Although there is very little difference in the separation plots for all three models, our conclusion is that model 1 (the Adorno thesis which emphasizes individual-level characteristics and personal attitudes) has the most predictive power given the level of concentration of events in the upper probabilities area in the plot.
```

Comment:
the ability of the models to predict the votes is more directly comparable.  See for example ways to compare predictions to actual outcomes which are used to evaluate weather forecasts: http://en.wikipedia.org/wiki/Brier_score